utils/connects.stanza

```diff
@@ -85,3 +91,23 @@ public defn connect-cp2105 (usb:JITXObject, uart:JITXObject) -> Instance  :
     net (uart.dtr, xcvr.e-uart.dtr)
     net (uart.rts, xcvr.e-uart.rts)
     xcvr
+
+; Connects a SPI controller to 1 or more peripherals, pulls gpio for peripherals[1..n] for cs
+public defn connect-spi (base:JITXObject, controller:JITXObject, peripherals:Tuple<JITXObject>) :
```

Comment:
why not splitting `peripherals:Tuple<JITXObject>` into 2: `peripheral-controller:JITXObject, peripherals-something:Tuple<<JITXObject>`

Reply:
the idea here is that it any combination of peripherals and controllers can be connected to a single controller as part of a bus